### DISCUSSION TITLE
.tool-versions at workspace level

### DISCUSSION THREAD

**Discussion by Arkoprabho** [dflook/terraform-github-actions#323] [Q&A] [answered]
Consider the following folder structure.
```
[]  .
├── .git
├── .github
├── .gitignore
├── .tool-versions
├── folder_a
└── folder_b
```
Where `folder_a` and `folder_b` contain terraform files.
How will say `terraform-plan` find the .tool-versions file given that the input `path` is set to `folder_a` | `folder_b`
Looking at code [here](https://github.com/dflook/terraform-github-actions/blob/977562aeb2756f7e8257bda745f423ffabb29813/image/src/terraform_version/asdf.py#L26) my understanding is that the code will continue to traverse up from the provided input_path up to the `workspace_path` but not the `workspace_path`.

Does this mean that a `.tool-version` in the root of the repo will get ignored?

**Comment by dflook** [comment, marked as answer]
Hi @Arkoprabho 👋  I can't remember the intention here, but looking at it now it looks like a defect. 

This is now working in v1.41.1

**Reply by Arkoprabho** [reply]
Thanks a ton
